geodocker-geoserver/Dockerfile.template

```diff
@@ -75,7 +75,37 @@ RUN set -x \
 RUN set -x \
     && rm /opt/tomcat/webapps/geoserver/WEB-INF/lib/guava-11.*.jar
 
+# Install Geomesa Kafka Distribution
+COPY --from=1 /geomesa-kafka* /opt/geomesa-kafka/
+
+# Install GeoMesa Kafka DataStore
+RUN set -x \
+  && cd /opt/tomcat/webapps/geoserver/WEB-INF/lib/ \
+  && tar zxvf /opt/geomesa-kafka/dist/gs-plugins/geomesa-kafka-${KAFKA_VERSION}-gs-plugin_2.11-${GEOMESA_VERSION}-install.tar.gz \
+  && rm /opt/tomcat/webapps/geoserver/WEB-INF/lib/zookeeper*.jar \
+  && /opt/geomesa-kafka/bin/install-kafka.sh /opt/tomcat/webapps/geoserver/WEB-INF/lib
+
+
+FROM quay.io/geomesa/base:__TAG__
+
+ARG GEOMESA_VERSION
+ARG ACCUMULO_VERSION
+ARG THRIFT_VERSION
+ARG KAFKA_VERSION
+
+ENV ACCUMULO_VERSION ${ACCUMULO_VERSION}
+ENV GEOMESA_VERSION ${GEOMESA_VERSION}
+ENV KAFKA_VERSION ${KAFKA_VERSION}
+ENV THRIFT_VERSION ${THRIFT_VERSION}
+ENV HADOOP_VERSION 2.7.3
+ENV GEOSERVER_VERSION 2.9.4
```

Comment:
this needs to be parameterized in tandem with geomesa...

Reply:
or be bumped if this docker is hard coding version 2.0 geomesa...should be 2.12.x